test/net/sourceforge/kolmafia/maximizer/MaximizerRegressionTest.java

```diff
@@ -109,6 +112,15 @@ public void keepsBonusBjornUnchanged() {
     assertEquals(25, modFor("Meat Drop"), 0.01);
   }
 
+  @Test
+  public void equipEmptyBjornNoSlot() {
+    equip(EquipmentManager.CONTAINER, "Buddy Bjorn");
+    KoLCharacter.addFamiliar(new FamiliarData(FamiliarPool.MOSQUITO));
+    KoLCharacter.setBjorned(FamiliarData.NO_FAMILIAR);
+    
+    assertTrue(maximize("item, -buddy-bjorn, +equip buddy bjorn"));
```

Comment:
I am lazy and could look things up myself but are "Buddy Bjorn", "buddy-bjorn" and "buddy bjprn" all supposed to be the same item?  Perhaps using the canonical name in all three places would be appropriate?

Reply:
`buddy-bjorn` refers to a slot, rather than an item, which is a good shout; I'll be consistent about the capitalization across the other two, and add a comment to that effect. Here, what's happening is that the buddy-bjorn slot is empty, and the maximizer is told not to fill that slot, so it refuses to equip the item entirely. Maybe this has to do with $familiar[none] not being the same as null, which I assume is what it defaults to. I can investigate it further, but the maximizer code is pretty labyrinthine to me.

Reply:
I think "buddy bjprn" is a typo on your end because I do not see it in my code, but I am not 1000% confident of that.

Reply:
Yes, typo.  The last parameter in "assertTrue(maximize("item, -buddy-bjorn, +equip buddy bjorn"));" since the name of the item was capitalized elsewhere.

"Here, what's happening is that the buddy-bjorn slot is empty, and the maximizer is told not to fill that slot, so it refuses to equip the item entirely" that sounds to me like correct behaviour.  I wonder if there is some "confusion" between using the back slot for the bjorn and changing the familiar in a bjorn that is already equipped?  Regardless I look forward to your next attempt.  Thank you.

Reply:
I'd like mafia to be willing to equip an empty bjorn because I plan on manually filling it myself. Because there's not a way to assign bonus values (or force-equips) to particular bjornified familiars, there aren't any other great wolutions. In particular, I don't want to constantly seesaw between mafia bjornifying a familiar and me replacing it basically once a turn, because that's two unnecessary server hits per adventure.

It feels like kind of a frivolous thing, but that's enough to make me feel like this is probably bad behavior--just possibly low-priority bad behavior. I've adjusted the capitalization, added comments, and disabled the test, so I'll undraft this now.